docs/release-notes/8.10.asciidoc

```diff
@@ -0,0 +1,43 @@
+[[release-notes-header-8.10.0]]
+== 8.10
+
+[discrete]
+[[release-notes-8.10.0]]
+=== 8.10.0
+
+[discrete]
+[[breaking-changes-8.10.0]]
+==== Breaking changes
+There are no breaking changes in 8.10.0.
+
+[discrete]
+[[features-8.10.0]]
+==== New features
```

Comment:
Leaving this open because still need a PR link and blog link from @PhilippeOberti and @paulewing.
 
```suggestion
==== New features
* Introduces a redesigned alert details flyout that enhances your investigative flows ({pull}TBD[#TBD]).
```

Reply:
Will be linking to doc PRs in lieu of kibana PRs here.

```suggestion
==== New features
* Introduces a redesigned alert details flyout that enhances your investigative flows (https://github.com/elastic/security-docs/pull/3816)[#3816], (https://github.com/elastic/security-docs/pull/3854)[#3854).
```